regulations/templates/regulations/layers/table.html

```diff
@@ -2,7 +2,7 @@
     Used to turn markdown tables into HTML tables
 {% endcomment %}
 <div class="table-wrap">
-    <table>
+    <table width="100%">
```

Comment:
Yes, commenting on this line in the template would help explain to future devs why it's here